components/blitz/src/ome/formats/importer/transfers/HardlinkFileTransfer.java

```diff
@@ -41,8 +41,9 @@
      * @param file File to be copied.
      * @param location Location to copy to.
      * @throws IOException
-     * @TODO Java7: replace ln once Java6 is dropped
+     *
      */
+    // TODO Java7: replace ln once Java6 is dropped
```

Comment:
Side-note: now Java6 is dropped on this line, we might want to make sure this is captured in a card for 5.2.0